### DISCUSSION TITLE
Projects need to be clear what version is created and current developed.

### DISCUSSION THREAD

**Discussion by DanielBitencourt** [godotengine/godot-proposals#6155] [Editor]
I had a bit of difficulty figuring out which version my old projects were created in, I found out by analyzing the project creation dates and the release dates of the Godot versions. I have now created a personal solution to solve this problem, for all of my new projects I will specify somewhere in the project or file the creation version, and also when I change versions in the same project.
Why isn't this basic and essential issue solved by Godot?

**Comment by Calinou** [comment]
This is already implemented in 4.0: https://github.com/godotengine/godot-proposals/issues/4355#issuecomment-1156451870

This solution won't be backported to `3.x`, as it involved some backwards-incompatible changes. It should however be possible to distinguish Godot 3.x projects from Godot 4.0 projects by looking at the `config_version` integer in `project.godot` (`4` is Godot 3.1+, `5` is Godot 4.0+).

**Reply by Zireael07** [reply]
I'm guessing he wants to distinguish between various 3.x versions?

**Reply by Calinou** [reply]
Unfortunately, this can't be done retroactively (other than distinguishing between 3.0 and 3.1+). That said, distinguishing between 3.x releases after 3.3 is less important because compatibility breakages between 3.3 and 3.5 are much less common than between 3.0 and 3.3.
